cards/portfolio/addon/templates/isolated.hbs

```diff
@@ -104,13 +98,20 @@
         <h3 class="network-section--title" data-test-portfolio-isolated-network-title>Ethereum Mainnet</h3>
         <span data-test-portfolio-isolated-network-asset-count>{{assets.length}} Assets</span>
       </header>
-      <ul class="network-section--grid">
-        {{#each sortedAssets as |asset index|}}
-          <li data-test-portfolio-asset={{index}}>
-            {{cardstack-content content=asset format="embedded"}}
-          </li>
-        {{/each}}
-      </ul>
+
+      {{#if isAscending}}
```

Comment:
@ef4 Have a question about ember-animated and wondering if you could share your opinion.

I had to repeat this `GridDisplay` component here only for animation reasons. Normally, I can sort and render list vs. grid-view of assets through CPs by just writing the component once: 
`<GridDisplay @contents={{sortedAssets}} @isListView={{isListView}} class={{if isListView "network-section--list"}} />`

However, this doesn't display any animation. The animation does work when I rerender the component according to each condition separately. Does this seem like the right approach?

Reply:
Oh yes, I see the problem.

When you pass `isListView` down as an argument, Ember is able to reuse all the components. There's no reason to destroy and recreate them. From Ember's perspective all that changes when `isListView` toggles is that one class name changes, and everything else stays the same.

That means the `animated-valued` down inside each card doesn't have anything to react to.

The way you solved it is a good workaround. It will cause the entire `<GridView>` to get destroyed and a new one created whenever `isListView` changes. 

Another way to achieve that would be:

```hbs
{{#animated-value isListView as |currentIsListView|}}
  <GridDisplay @contents={{sortedAssets}} @isListView={{currentIsListView}} class={{if currentIsListView "network-section--list"}} data-test-network-section-grid />
{{/animated-value}}
```

This also works, although it may have other unintended side-effects. If there was content below this, you'd probably need another `<AnimatedContainer/>` around this. And it's a bit overkill to have an `animated-value` here when we don't actually intend it to animate. This makes me think I should add a component to ember-animated specifically for telling descendant animators to treat a given change as a reason to animate.